repository/meteorRepository.js

```diff
@@ -0,0 +1,14 @@
+import axios from "axios";
+import { NASA_API_KEY, NASA_API_URL } from "../config.js";
+
+const seventhOfOctober = "2024-10-07";
+const eleventhOfOctober = "2024-10-11";
```

Comment:
best practices for naming constants is using UPPERCASE f.e.` const SEVENTH_OF_OCTOBER = ""` (can fix in next PRs)